src/test/java/edu/kis/vh/nursery/RyhmersJUnitTest.java

```diff
@@ -3,38 +3,38 @@
 import org.junit.Assert;
 import org.junit.Test;
 
+import edu.kis.vh.nursery.stack.IntArrayStack;
+
 public class RyhmersJUnitTest {
 
+	final int EMPTY_STACK_VALUE = 0;

```

Comment:
Original constant should be used (it is better to use a constant from stacks interface).